oecophylla/taxonomy/taxonomy.sh

```diff
@@ -17,19 +17,10 @@ source activate oecophylla-shogun
 
 echo $CONDA_PREFIX
 
-mkdir -p $CONDA_PREFIX/etc/conda/activate.d
-mkdir -p $CONDA_PREFIX/etc/conda/deactivate.d
+cd $CONDA_PREFIX/bin
 
-touch $CONDA_PREFIX/etc/conda/activate.d/env_vars.sh
-touch $CONDA_PREFIX/etc/conda/deactivate.d/env_vars.sh
+wget https://github.com/knights-lab/UTree/releases/download/v2.0c/utree-search_gg
+chmod 755 utree-search_gg
 
-# path to the pre-compiled binaries on Barnacle
-bindir=/projects/ag500/shogun/bin
-
-echo '#!/bin/sh' >> $CONDA_PREFIX/etc/conda/activate.d/env_vars.sh
-echo "export OLD_PATH=$PATH" >> $CONDA_PREFIX/etc/conda/activate.d/env_vars.sh
-echo "export PATH=$PATH:$bindir" >> $CONDA_PREFIX/etc/conda/activate.d/env_vars.sh
-
-echo '#!/bin/sh' >> $CONDA_PREFIX/etc/conda/deactivate.d/env_vars.sh
-echo "export PATH=$OLD_PATH" >> $CONDA_PREFIX/etc/conda/deactivate.d/env_vars.sh
-echo "unset OLDPATH" >> $CONDA_PREFIX/etc/conda/deactivate.d/env_vars.sh
+wget https://github.com/knights-lab/BURST/releases/download/v0.99.4a/burst_linux_DB15
```

Comment:
Just to clarify, the most recent shogun conda recipes have all of the paths configured appropriately?

Reply:
Yep, Gabe pushed the appropriate binaries to GitHub, so we don't need to have any Barnacle-specific hacks.

Reply:
That's good news!